style.css

```diff
@@ -0,0 +1,32 @@
+* {
+ margin: 0;
+ padding: 0;
+ width: 100%;
+ box-sizing: border-box;
+ border-style: 1px solid black;
+}
+
+html, body {
+ box-sizing: border-box;
+ border-style: 1px solid black;
+ height: 100%;
+ width: 100%;
+}
+
+.fa {
+ float: right;
+ font-size: 3em;
+}
+
+.main-nav {
+  padding: 0.1em;
+  font-size: 1.5em;
+  float:left;
+  width: 960px;
+}
+
+@media screen and (min-width:640px) {
```

Comment:
with a min-width, you are saying that the styles below will kick-in when the screen becomes more than 640px wide. This means that this is not a mobile-first webpage.